src/Auth/authSaga.js

```diff
@@ -0,0 +1,8 @@
+import { select, take } from 'redux-saga/effects';
+import { LOGIN_REQUEST } from './authModule';
+
+export default function* loginFlow() {
+  yield take(LOGIN_REQUEST);
+  const { authModule } = yield select();
```

Comment:
Try to write a test for it. Then you will know. Also `` localStorage `` makes it difficult to test too.